### DISCUSSION TITLE
pls add storage unit

### DISCUSSION THREAD

**Discussion by Smallkittyy** [danielkrupinski/Osiris#3206] [Ideas]
Could you add the Storage uint to the inv chnager?
i think im not alone for ppl who open cases for stat trak guns. 

and a storage Unit is a good way to get rid of many skins (deleting them in the inv changer option takes really really long and is anoying if you remove the wrong gun by accident)

would be great if you can add that :D

here a ice for the hot weather 🍧

**Comment by 0killdude** [comment]
you can get stat trak guns and change wear in config files

**Reply by Smallkittyy** [reply]
no you cant add stat trak guns in the inv changer.. you can only add the gun and how many of them.

**Reply by 0killdude** [reply]
see after adding the guns you should go to the config file then under  "Wear": 0.000458 you should write this "StatTrak": 0 then your weapon will become stat trak remember to give Ctrl + S tho so the config file is saved, then open the game and enjoy your Stat Trak weapons
![stattrak](https://user-images.githubusercontent.com/88573461/130166081-3ce3ec34-cf70-4e03-a6df-1141bb9862b9.png)

**Reply by Smallkittyy** [reply]
thanks

**Reply by Smallkittyy** [reply]
> see after adding the guns you should go to the config file then under "Wear": 0.000458 you should write this "StatTrak": 0 then your weapon will become stat trak remember to give Ctrl + S tho so the config file is saved, then open the game and enjoy your Stat Trak weapons
> ![stattrak](https://user-images.githubusercontent.com/88573461/130166081-3ce3ec34-cf70-4e03-a6df-1141bb9862b9.png)

how do i know which skin is which? there is no weapon/skin name... (if i make the skins stat trak the skins get removed!)

**Reply by KurastikO** [reply]
name them?

**Reply by Smallkittyy** [reply]
> name them?

i found a skin ID list, but if i edit the cfg all skins jsut get removed xD

**Reply by KurastikO** [reply]
![image](https://user-images.githubusercontent.com/68241623/130353081-3cdde86b-0612-4963-9803-c9e6ecf31164.png)

**Reply by danielkrupinski** [reply]
@KurastikO I've added weapon and paint kit names to config (1a2c983c472bdb128f8f6a06ad9ac5d9d420763a & e98bc7b6b7cac1b6fda759e444eaf4d133cc461f)

**Reply by KurastikO** [reply]
thanks daniel
